code/game/machinery/bots/medbot.dm

```diff
@@ -559,7 +599,20 @@
 		A.skin = "tox"
 	else if(istype(src,/obj/item/weapon/storage/firstaid/o2))
 		A.skin = "o2"
-
+	else if(istype(src,/obj/item/weapon/storage/firstaid/brute))
+		A.skin = "brute"
+	else if(istype(src,/obj/item/weapon/storage/firstaid/adv))
+		A.skin = "adv"
+	else if(istype(src,/obj/item/weapon/storage/firstaid/tactical))
+		A.skin = "bezerk"

```

Comment:
Oh, I don't doubt that it's a typo---but as far as the icon name? It's been "bezerk" for years (and reference in a few sports as this)--thus why it is what it is *shrug*.